starter-code/pizza.js

```diff
@@ -1 +1,85 @@
-// Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+
+
+  //Buttons
+  var counter = 16
+
+  $("button").click(function(event){
+    $(this).toggleClass("active");
+  });
+
+
+
+  $("button.btn-pepperonni").click(function(event){
+    $("section.pep").toggle("display");
+    $("li:contains('pepperonni')").toggle("display");
+    if(!$("button.btn-pepperonni").hasClass("active")){
+      $(".price strong").text("$" + (--counter));
+    } else {
+      $(".price strong").text("$" + (++counter));
+    }
+
+  });
+
+
+
+
+  $("button.btn-green-peppers").click(function(event){
+    $("section.green-pepper").toggle("display");
+    $("li:contains('green peppers')").toggle("display");
+    if(!$("button.btn-green-peppers").hasClass("active")){
+      $(".price strong").text("$" + (--counter));
+    } else if ($("button.btn-green-peppers").hasClass("active")){
+      $(".price strong").text("$" + (++counter));
+    }
+  });
+
+  $("button.btn-mushrooms").click(function(event){
+    $("section.mushroom").toggle("display");
+    $("li:contains('mushrooms')").toggle("display");
+    if(!$("button.btn-mushrooms").hasClass("active")){
+      $(".price strong").text("$" + (--counter));
+    } else if ($("button.btn-mushrooms").hasClass("active")){
+      $(".price strong").text("$" + (++counter));
+    }
+  });
+
+  //Sauce and crust Buttons
+
+
+  //Sauce
+  $("button.btn-sauce").toggleClass("active");
+  $("section.sauce").removeClass("sauce-white");
+
+  $("button.btn-sauce").click(function(event){
+    $("section.sauce").toggleClass("sauce-white");
+    $("li:contains('white sauce')").toggle("display");
+  });
+
+  if (!$("button.btn-sauce").hasClass("active")){
+    $("li:contains('white sauce')").css({"display":"none"});
+  }
+
+
+  // Crust
+  $("button.btn-crust").toggleClass("active");
+  $("section.crust").removeClass("crust-gluten-free");
+
+  $("button.btn-crust").click(function(event){
+    $("section.crust").toggleClass("crust-gluten-free");
+    $("li:contains('gluten-free crust')").toggle("display");
+  });
+
+  if (!$("button.btn-crust").hasClass("active")){
+    $("li:contains('gluten-free crust')").css({"display":"none"});
+  }
+
+
+  //Adding to the final price
+
+  
+   $(".price strong").text("$" + counter);
+
+
+});
+/* faltaria sumar y restar  el valor de btn crust y el de btn-sauce per al ser  un valor expecifico no se como hacerlo*/
```

Comment:
Tendríais dos opciones para hacerlo, la primera es meter a machete el valor que queréis que sume o reste en el momento de dar al botón (es decir, igual que habéis hecho al principio con los botones del peperonis, los pimientos verdes y los champiñones). Pero no sería lo más correcto, ya que si alguien quiere cambiar el precio en el html, tendría que hacerlo también en el código, y éste debería cambiar dinámicamente los valores en función de los que  haya representados en el DOM. Para ello deberíais coger los valores de los elementos de la lista que hay en el DOM con la clase `panel price` y etiqueta `aside`.